content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -52,7 +52,7 @@ User namespaces is a Linux feature that allows to map users in the container to
 different users in the host. Furthermore, the capabilities granted to a pod in
 a user namespace are valid only in the namespace and void outside of it.
 
-A pod can opt-in to use user nameapces by setting the `pod.spec.hostUsers` field
+A pod can opt-in to use user namespaces by setting the `pod.spec.hostUsers` field
```

Comment:
I think it's more clear to fix the typo in another PR.

Reply:
typo fix here: https://github.com/kubernetes/website/pull/36417